ci/prow/config_knative.yaml

```diff
@@ -73,6 +73,12 @@ presubmits:
       args:
       - "--run-test"
       - "./test/e2e-tests.sh --gloo-version 0.17.1"
+    - custom-test: kourier-stable
+      always_run: false
+      optional: true
+      args:
+        - "--run-test"
+        - "./test/e2e-tests.sh --kourier stable"
```

Comment:
You have different flags here: --kourier here and --kourier-version in line 295.

Reply:
I see that the correct flag is --kourier-version: https://github.com/knative/serving/blob/master/test/e2e-common.sh#L100

Reply:
```suggestion
        - "./test/e2e-tests.sh --kourier-version stable"
```

Reply:
good catch! fixed, thanks